src/google/protobuf/stubs/map_util.h

```diff
@@ -156,11 +156,11 @@ FindOrNull(Collection& collection,  // NOLINT
 }
 
 // Returns the pointer value associated with the given key. If none is found,
-// NULL is returned. The function is designed to be used with a map of keys to
+// nullptr is returned. The function is designed to be used with a map of keys to
 // pointers.
 //
 // This function does not distinguish between a missing key and a key mapped
-// to a NULL value.
+// to a nullptr value.
```

Comment:
It sounds kind of strange to say "a nullptr value"--maybe replace that with just "nullptr"?

Reply:
Ack